psse/psse-converter/src/main/java/com/powsybl/psse/converter/PsseImporter.java

```diff
@@ -242,7 +353,7 @@ private static void createGenerator(PsseGenerator psseGen, ContainersMapping con
     }
 
     private static void createBuses(PsseRawModel psseModel, ContainersMapping containerMapping, PerUnitContext perUnitContext,
-                                    Network network) {
+                                    Network network, HashMap<Integer, PsseBus>  busNumToPsseBus) {
```

Comment:
It's better to pass Map (interface), except if you need a method that is provided only by this implementation. (global to the whole file)